ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -820,14 +820,14 @@ export default defineComponent({
           const today = new Date()
           const expireDate = new Date()
           // expireDate.setDate(expireDate.getDate() + days)
-          var dateExpiry = moment(new Date(
+          var dateExpiry = moment.utc(new Date(
```

Comment:
This is the change I made (+ unit test update) for fix the dates issue. @cameron-eyds

Reply:
I'm not fully familiar with moment or the implications but if you are confident than so am i :)

Reply:
I verified that before and after this change, the number of days to expire was the same. Also, we are using UTC format in the `Date()` inside the moment function.